api/python/provisioner/commands/confstore_export.py

```diff
@@ -54,6 +55,24 @@ class ConfStoreExport(CommandParserFillerMixin):
     input_type: Type[inputs.NoParams] = inputs.NoParams
     description = "Export pillar template data to ConfStore"
 
+    def _encrypt_value(self, key, value):
+        from cortx.utils.security import cipher
+        if not value:
+            return value
+        cypher_name = key.split('>')[0]
+        if 'bmc' in key:
```

Comment:
For future improvements: instead of multiple `if-elif-esle` blocks we can suggest using for parsing some schemes independent from names like `bmc` and `storage_enclosure`.

Reply:
Here - EOS-19150 few things have been listed down which would go as improvement for this.
There i will incorporate these changes.

Reply:
Will be resolved in
https://jts.seagate.com/browse/EOS-19150